src/libbson/tests/json/bson_corpus/top-mongoc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Top-level document validity - libmongoc-specific tests",
```

Comment:
Noted that this file will eventually be relocated for [CDRIVER-4016](https://jira.mongodb.org/browse/CDRIVER-4016).

Reply:
Thanks for the link! Updated the description to note CDRIVER-4016.